### ISSUE TITLE
2 variables "c" declared as char should be int?

### ISSUE THREAD

**Issue by tonycurtis** [openucx/ucx#340]
Both used with getopt()

src/tools/perf/perftest.c @ 466

```
    458 static ucs_status_t read_batch_file(FILE *batch_file, ucx_perf_params_t 
    458 *params,
    459                                     char** test_name_p)
    460 {
    461 #define MAX_SIZE 256
    462     ucs_status_t status;
    463     char buf[MAX_SIZE];
    464     int argc;
    465     char *argv[MAX_SIZE + 1];
    466     char c, *p;
    467 
```

src/ucs/stats/stats_reader.c @ 224
```
    219 int main(int argc, char **argv)
    220 {
    221     GtkWidget *window;
    222     ucs_status_t status;
    223     int port;
    224     char c;
```

**Comment by shamisp** [comment]
Do you want to submit a patch ?

**Comment by yosefe** [comment]
fixed in #342
